src/components/accordion-item/daikin-accordion-item.ts

```diff
@@ -142,9 +149,6 @@ export class DaikinAccordionItem extends LitElement {
   @property({ type: Boolean, reflect: true })
   disabled = false;
 
-  @query("summary")
-  private _summary!: HTMLElement | null;
-
   /**
    * Actual presence of `open` attribute of the `<details>` element.
```

Comment:
```suggestion
   * Whether or not the animation is being performed.
```

Please delete the rest of the comments.